Sources/Run/Commands.swift

```diff
@@ -84,6 +84,38 @@ final class TestPayments: Command, ConfigInitializable {
     }
 }
 
+final class TestShopping: Command, ConfigInitializable {
+    public let id = "do_shopping"
+    public let help = ["This command does shopping experience"]
+    public let console: ConsoleProtocol
+    private let fbId = fbIdLilia
```

Comment:
p2: fbId will be defferent for staging or development. It will be good to use arguments in function run to get fbId when call command entered. It will make test functions universal for any developer. This is not required to do.